### PR TITLE
export all style consts from this package

### PR DESCRIPTION
**Pull request by willisplummer**
moves all of our styling constants into this file + does a little renaming to hopefully make them easier to access.

one challenge is that it's not great to access an object with a number key so i renamed some of the styles.. curious about alternative naming schemes + opinions etc.

### CODE REVIEW

File: src/constants/border-radius.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+// @flow
+
+import { css } from 'styled-components';
+
+const F1 = css`
```
Comment (iansolano) [inline src/constants/border-radius.js]: why are these named `F(n)`?

File: src/constants/font-weight.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+// @flow
+
+import { css } from 'styled-components';
+
+export const FW_700 = css`
```
Comment (iansolano) [inline src/constants/font-weight.js]: why are these exported?

File: src/constants/border-radius.js (hunk 1 above)
Comment (willisplummer) [reply src/constants/border-radius.js]: you can't name a const starting with a number.. wasn't sure about a coherent naming convention

File: src/constants/font-weight.js (hunk 2 above)
Comment (willisplummer) [reply src/constants/font-weight.js]: yeah shouldn't be

File: src/constants/border-radius.js (hunk 1 above)
Comment (willisplummer) [reply src/constants/border-radius.js]: does `BR_1` type naming make sense to you?

Comment (willisplummer) [reply src/constants/border-radius.js]: also curious about your thoughts re standarding `BR1` vs `BR_1` (for example fontweights and borderwidths are `BW_` and `FW_` but some of the other consts where it's a single letter are just `X1` e.g.

**Comment by iansolano** [comment]
LGTM, my only comment is - about consistency with naming? i.e. style initials then number, b/c at the moment we name things differently (sometimes with underscores separating number, sometimes single words, style initials, etc).

File: src/constants/border-radius.js (hunk 1 above)
Comment (iansolano) [reply src/constants/border-radius.js]: yeah, just made a general comment about this. i don't really care which way, as long as we pick one. my vote would be for using `_` just b/c for my it's easier to read / scan for in a file but tbh cool either way

**Comment by willisplummer** [comment]
@iansolano standardized the naming.. 👍 ?

**Comment by iansolano** [comment]
:shipit:

**Pull request merged**